prometheus/filter_test.go

```diff
@@ -0,0 +1,133 @@
+package prometheus_test
```

Comment:
Would it be possible to get some examples here like [these](https://blog.golang.org/examples). I think it would help a lot with the intention of how filters are intended to be used.l

Reply:
Here is the commit: https://github.com/influxdata/influxdb/pull/11347/commits/6a87e2ae99b77f6ca063704c56109ba65881986b#diff-81c1daf4c558db6b0e058827675ae5e3R1